frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -476,15 +477,25 @@ const QueueItem: any = ({
       </li>
     ) : null;
 
+  const timer = useTestTimer(internalId);
```

Comment:
Yeah I was thinking we'd need to hoist this up in order to color the queue items. Didn't turn out that bad!